lib/ansible/modules/cloud/amazon/GUIDELINES.md

```diff
@@ -158,11 +159,45 @@ caught, you should catch `ClientError` and `BotoCoreError` exceptions with every
 See exception handling. module_utils.ec2 checks for missing profiles or a region not set when it needs to be,
 so you don't have to.
 
+```python
```

Comment:
This is very cool, I'd missed this somehow. Can we move this nearer to your new line 129 to make it more obvious?